### PR TITLE
quickstart: Simplify generated conf.py (HTML)

### PR DESCRIPTION
**Pull request by tk0miya**
### Feature or Bugfix
- Refactoring

### Purpose
- refs: #4256

### CODE REVIEW

File: sphinx/templates/quickstart/conf.py_t [hunk 1]
```diff
@@ -76,32 +76,11 @@ exclude_patterns = [{{ exclude_patterns }}]
...
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-#
-# html_theme_options = {}
-
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['{{ dot }}static']
```
Comment (tk0miya) [inline sphinx/templates/quickstart/conf.py_t:97]: I consider to change the default value of `html_static_path` also. Then we can omit whole of "Options for HTML output" section by default.

**Comment by tk0miya** [comment]
I don't know these options should be removed or not. So I need for your comment.

File: sphinx/templates/quickstart/conf.py_t (hunk 1 above)
Comment (shimizukawa) [reply sphinx/templates/quickstart/conf.py_t:97]: +1 to remove `html_static_path` configuration.

File: sphinx/templates/quickstart/conf.py_t [hunk 2]
```diff
@@ -76,32 +76,11 @@ exclude_patterns = [{{ exclude_patterns }}]
...
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = 'alabaster'
-
-# Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
-#
-# html_theme_options = {}
```
Comment (shimizukawa) [inline sphinx/templates/quickstart/conf.py_t:98]: +1 to remove `html_theme_options `

File: sphinx/templates/quickstart/conf.py_t [hunk 3]
```diff
@@ -76,32 +76,11 @@ exclude_patterns = [{{ exclude_patterns }}]
...
 
-# Custom sidebar templates, must be a dictionary that maps document names
-# to template names.
-#
-# The default sidebars (for documents that don't match any pattern) are
-# defined by theme itself.  Builtin themes are using these templates by
-# default: ``['localtoc.html', 'relations.html', 'sourcelink.html',
-# 'searchbox.html']``.
-#
-# html_sidebars = {}
```
Comment (shimizukawa) [inline sphinx/templates/quickstart/conf.py_t:113]: +1 to remove `html_sidebars`

File: sphinx/templates/quickstart/conf.py_t [hunk 4]
```diff
@@ -76,32 +76,11 @@ exclude_patterns = [{{ exclude_patterns }}]
 
 # -- Options for HTML output -------------------------------------------------
 
-# The theme to use for HTML and HTML Help pages.  See the documentation for
-# a list of builtin themes.
-#
-html_theme = 'alabaster'
```
Comment (shimizukawa) [inline sphinx/templates/quickstart/conf.py_t]: +0 to remove `html_theme` because I often changed this option after `sphinx-quickstart`.

**Review by shimizukawa — APPROVED** [review]
LGTM

File: sphinx/templates/quickstart/conf.py_t (hunk 4 above)
Comment (tk0miya) [reply sphinx/templates/quickstart/conf.py_t]: Okay, let's left not removed it.

File: sphinx/builders/html.py [hunk 5]
```diff
@@ -1403,7 +1403,7 @@ def setup(app):
     app.add_config_value('html_favicon', None, 'html', [str])
     app.add_config_value('html_css_files', [], 'html')
     app.add_config_value('html_js_files', [], 'html')
-    app.add_config_value('html_static_path', [], 'html')
+    app.add_config_value('html_static_path', ['_static'], 'html')
```
Comment (tk0miya) [inline sphinx/builders/html.py]: I modified the default value of `html_static_path` to remove the definition from `conf.py`.

File: sphinx/templates/quickstart/conf.py_t [hunk 6]
```diff
@@ -53,9 +53,11 @@ extensions = [
 {%- endfor %}
 ]
 
+{% if dot != '_' -%}
 # Add any paths that contain templates here, relative to this directory.
 templates_path = ['{{ dot }}templates']
```
Comment (tk0miya) [inline sphinx/templates/quickstart/conf.py_t]: I also removed `templates_path` by default. This is very similar to the case of `html_static_path` to me.

**Review by tk0miya — COMMENTED** [review]
@shimizukawa Thank you for reviewing. updated.

File: sphinx/builders/html.py (hunk 5 above)
Comment (tk0miya) [reply sphinx/builders/html.py]: OMG, this settings force to users to create `_static` directory on Sphinx project even if they does not use quickstart...
That must not be good. I'll revert this change ASAP.

**Pull request merged**